### ISSUE TITLE
Requires spooler

### ISSUE THREAD

**Issue by ajostergaard** [unbit/django-uwsgi#3]
Hello,

It seems like django-uwsgi requires a spooler to be defined:

KeyError at /admin/uwsgi/
'spooler'

Django Version:	1.8.4
Exception Value:	'spooler'
Exception Location:	/root/python-envs/django184/lib/python2.7/site-packages/django_uwsgi/stats.py in get_uwsgi_stats, line 17

/root/python-envs/django184/lib/python2.7/site-packages/django_uwsgi/stats.py in get_uwsgi_stats
    jobs = []
                if uwsgi.opt['spooler']: ...
        spooler_jobs = uwsgi.spooler_jobs()

Is that intentional?

Thanks,
AJ

**Comment by laike9m** [comment]
Same here:

```
KeyError at /
'spooler'
Request Method:	GET
Request URL:	https://laike9m.com/
Django Version:	1.8.5
Exception Type:	KeyError
Exception Value:	
'spooler'
Exception Location:	/home/laike9m/Envs/blog/lib/python3.4/site-packages/django_uwsgi/stats.py in get_uwsgi_stats, line 17
Python Executable:	/usr/local/bin/uwsgi
Python Version:	3.4.3
Python Path:	
['.',
 '',
 '/home/laike9m/Envs/blog/lib/python3.4',
 '/home/laike9m/Envs/blog/lib/python3.4/plat-x86_64-linux-gnu',
 '/home/laike9m/Envs/blog/lib/python3.4/lib-dynload',
 '/usr/lib/python3.4',
 '/usr/lib/python3.4/plat-x86_64-linux-gnu',
 '/home/laike9m/Envs/blog/lib/python3.4/site-packages']
Server time:	Sun, 11 Oct 2015 00:05:40 +0800
```

**Comment by ajostergaard** [comment]
@laike9m - fortunately it's fairly easy to configure a spooler and doesn't seem to have any real down-side. Of course at a minimum the dependency should be documented.

**Comment by Budulianin** [comment]
Actual in my case

**Comment by MechanisM** [comment]
Fixed in #5 thanks to @alanjds
